### ISSUE TITLE
Allow renaming of fixtures-as-arguments?

### ISSUE THREAD

**Issue by Sushisource** [la10736/rstest#107]
Thanks for this crate, super useful.

I find myself often wanting to have a long, specific name for a fixure, but a much shorter name when actually using it in the test. It'd be nice to be able to rename fixtures in the `#[rstest]` macro for that purpose.

For example:
```rust
#[rstest(long_setup_name("arg") as foo)]
fn test_name(foo: FakeFoo) {
}
```

Would this be welcome? I would be happy to look into making a PR if so.

**Comment by la10736** [comment]
Great idea!!! Sure it's welcome!

Thx

**Comment by la10736** [comment]
Released in 0.10.0

**Comment by Sushisource** [comment]
@la10736 Awesome, thank you! Never was able to find the time to try to contribute it :sweat_smile:
